imports/services/BoardService.ts

```diff
@@ -79,4 +83,35 @@ export class BoardService extends BaseService {
     }
     return board
   }
+
+  public async starBoard(boardId: string): Promise<boolean> {
+    const userId = Meteor.userId()
+    if (userId != null) {
+      return BoardFavoritesCollection.addFavoriteBoard(userId, boardId)
+    } else {
+      throw new Meteor.Error('Cannot Star the board')
+    }
+  }
+
+  public cancelStarBoard(boardId: string): Promise<boolean> {
+    const userId = Meteor.userId()
+    if (userId != null) {
+      return BoardFavoritesCollection.removeFavoriteBoard(userId, boardId)
+    } else {
+      throw new Meteor.Error('Cannot Unstar the board')
+    }
+  }
+
+  public async getMineFavoriteBoards(): Promise<IBoard[]> {
+    const userid = Meteor.userId()
+    if (userid != null) {
+      const favoriteBoardRecords = await BoardFavoritesCollection.getFavoriteBoards(userid)
+      const favoriteBoards = favoriteBoardRecords.map(record =>
+        this._makeBoard(record, { keys: ['id', 'title'] })
```

Comment:
there's `_makeBoards` methods in the `BoardsSubscribers` already, maybe we could move it into a utils functions in a seperate file, we can use it in both sides?